There is a unit test failing in `queryUtils`. Could you please take a look at this?
Also a linting error:
```
src/public/app/plugins/datasource/influxdb/influx_query_model.ts
  166:78  error  Expected '!==' and instead saw '!='  eqeqeq
```

--- a/public/app/plugins/datasource/influxdb/influx_query_model.ts
+++ b/public/app/plugins/datasource/influxdb/influx_query_model.ts
@@ -163,7 +163,7 @@ export default class InfluxQueryModel {
       if (interpolate) {
         value = this.templateSrv.replace(value, this.scopedVars);
       }
-      if (!operator.startsWith('>') && !operator.startsWith('<') && operator != '<>') {
+      if ((!operator.startsWith('>') && !operator.startsWith('<')) || operator === '<>') {
         value = "'" + value.replace(/\\/g, '\\\\').replace(/\'/g, "\\'") + "'";
       }
     } else if (interpolate) {

